subsys/net/Kconfig

```diff
@@ -85,12 +85,18 @@ config  NETWORKING
 	select NET_BUF
 	select POLL
 	select ENTROPY_GENERATOR
+	select COOP_ENABLED
```

Comment:
I don't think this actually does anything useful.

You can test this by trying to come up with a situation where this select prevents an invalid configuration from occuring.